### PR TITLE
Add getItemEnchantmentLevel() function, to easily see if an item has an enchantment and at what level.

### PR DESCRIPTION
**Pull request by zisis912**
title

### CODE REVIEW

**Comment by extremeheat** [comment]
This seems unnecessary when you can do this in one line already with .find

```js
item.enchants.find(e => e.name === 'efficiency')?.level
```

**Comment by extremeheat** [comment]
In this case `item.enchants` likely would have been better as a dictionary (since enchant.name is a unique key), but not a big deal.

You can get a dict yourself with `Object.fromEntries(item.enchants.map(e => [e.name, e]))`

**Comment by extremeheat** [comment]
Ah, one thing I did realize with the current implementation is that it has a enchants setter (these are bad), so to edit the enchants, you have to 

```js
const enchants = item.enchants
enchants.find(ench => enc.name === 'efficiency').level = 3
item.enchants = enchants
```

IMO all the get/set inside prismarine-item should be removed for non-obvious usage patterns like this. `item.enchants` returns a mutation, so you need to store to a variable else simply editing the object and trying to trigger the setter again will simply edit a GC'ed item.

**Comment by zisis912** [comment]
I made this commit because I am translating large parts of the java source code to mineflayer to implement more clicking tyoes, so having identically named functions is a big help.

Also in my opinion both setters and getters should not be a thing, they are both stupid and serve no purpose. I would choose `getEnchants()` and `setEnchants` any day

File: index.js [hunk 1]
```diff
@@ -158,6 +158,14 @@ function loader (registryOrVersion) {
       throw new Error("Don't know how to deserialize for this mc version ")
     }
 
+    getItemEnchantmentLevel (enchantName) {
+      if (!this.enchants) return 0
+      for (const enchant of this.enchants) {
```
Comment (rom1504) [inline index.js:163]: enchants should be a dict if you want that

File: index.d.ts [hunk 2]
```diff
@@ -24,6 +24,7 @@ export class Item {
   get blocksCanDestroy(): [string][];
   set blocksCanDestroy(blockNames: string[]);
   repairCost: number;
+  getItemEnchantmentLevel(enchantName: string): number
```
Comment (rom1504) [inline index.d.ts:27]: can you update the doc

**Comment by rom1504** [comment]
old, please re open to finish

**Pull request closed without merging**